cilium/cmd/debuginfo.go

```diff
@@ -41,13 +45,26 @@ const (
 	STDOUT outputType = 0 + iota
 	MARKDOWN
 	HTML
+	JSONOUTPUT
+	JSONPATH
+)
+
+var (
+	// Can't tall it jsonOutput because another var in this package uses that.
```

Comment:
tall is call, perhaps?

Reply:
FYI I refactor the other variable out in https://github.com/cilium/cilium/pull/5873

Reply:
I saw it when it was filed, but forgot to address. Will address in next upload.

Reply:
Fixed in next upload.